src/main/charts/bamboo/values.yaml

```diff
@@ -568,6 +568,10 @@ bamboo:
     #
     type: ClusterIP
 
+    # -- NodePort for Bamboo service
```

Comment:
Do we need to specify that this is applied only when the service is a type of `NodePort` or is this common pattern?